docs/src/main/sphinx/release/release-423.md

```diff
@@ -0,0 +1,134 @@
+# Release 423 (9 Aug 2023)
+
+## General
+
+* Add support for renaming nested fields in a column with a `RENAME COLUMN`
+  statement. ({issue}`16757`)
+  * Add support for setting the type of a nested field in a column with a `SET
+  DATA TYPE` statement. ({issue}`16959`)
+* Add support for column comments on materialized views. ({issue}`18016`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18212`)
+* Improve performance when planning queries involving multiple window functions. ({issue}`18491`)
+* Improve optimizer estimates for queries with a `BETWEEN` clause. ({issue}`18501`)
```

Comment:
What's the user-visible effect? Maybe rephrase to:

> Improve performance of queries involving `BETWEEN` clauses.

Reply:
yeah .. that sounds good. Done.